Sources/MongoSwift/ChangeStream.swift

```diff
@@ -3,12 +3,17 @@ import mongoc
 /// `ChangeStreamOptions` to resume or start a change stream where a previous one left off.
 /// - SeeAlso: https://docs.mongodb.com/manual/changeStreams/#resume-a-change-stream
 public struct ResumeToken: Codable, Equatable {
-    private let resumeToken: Document
+    public let resumeToken: Document
 
-    internal init(resumeToken: Document) {
+    internal init(_ resumeToken: Document) {
         self.resumeToken = resumeToken
     }
 
+    public func encode(to encoder: Encoder) throws {
+        var container = encoder.singleValueContainer()
+        try container.encode(self.resumeToken)
+    }
+
     public init(from decoder: Decoder) throws {
         self.resumeToken = try Document(from: decoder)
```

Comment:
nit: while this works, I think it would be better to write it using a `SingleValueDecodingContainer` so it's symmetric to the `encode` method:
```swift
let container = try decoder.singleValueContainer()
self.resumeToken = try container.decode(Document.self)
```